bin/gcs-deploy.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
```

Comment:
Could we move this script to `.utils/` like the other "ops scripts"?

Reply:
Oh, I think that's fine in `.utils/` still, or make it `.utils/gcs/` if you want.

Reply:
Just moved it to .utils to keep things consistent